packages/components/src/popover/stories/index.story.tsx

```diff
@@ -255,3 +258,76 @@ export const WithSlotOutsideIframe: StoryObj< typeof Popover > = {
 		...Default.args,
 	},
 };
+
+export const WithCloseHandlers: StoryObj< typeof Popover > = {
+	render: ( args ) => {
+		const WithHandlers = () => {
+			const [ isVisible, setIsVisible ] = useState( false );
+			const buttonRef = useRef< HTMLButtonElement >( null );
+
+			const toggleVisible = ( event: React.MouseEvent ) => {
+				if ( buttonRef.current && event.target !== buttonRef.current ) {
+					return;
+				}
+				setIsVisible( ( prev ) => ! prev );
+			};
+
+			const handleClose = () => {
+				setIsVisible( false );
+			};
+
+			const handleFocusOutside = () => {
+				setIsVisible( false );
+			};
```

Comment:
Let's call `args.onClose?.()` and `args.onFocusOutside?.( e )` respectively, which will allow Storybook actions to log as expected.